pkgdb2/api/packages.py

```diff
@@ -905,22 +905,29 @@ def api_package_list(pattern=None):
         critpath = False
     elif critpath:
         critpath = True
-    acls = bool(flask.request.args.get('acls', False))
+    acls = flask.request.args.get('acls', False)
+    if str(orphaned).lower() in ['0', 'false']:
```

Comment:
for tech-debt-fighting week, one task could be to convert all little blocks like this to use an ``asbool`` function or something like that.

```python
acls = asbool(flask.request.args.get('acls', False))

def asbool(value):
    if value is None:
        return False
    if isinstance(value, bool):
        return value
    return str(value).lower() in ['1', 'true', 'yes', 'please']  # etc...